scripts/tree_count.json

```diff
@@ -0,0 +1,198 @@
+                        "name":"trnk_other",
+                        "type":"char"
+                    },
+                    {
+                        "name":"brnch_ligh",
+                        "type":"char"
+                    },
+                    {
+                        "name":"brnch_shoe",
+                        "type":"char"
+                    },
+                    {
+                        "name":"brnch_othe",
+                        "type":"char"
+                    },
+                    {
+                        "name":"address",
+                        "type":"char"
+                    },
+                    {
+                        "name":"zipcode",
+                        "type":"int"
+                    },
+                    {
+                        "name":"zip_city",
+                        "type":"char"
+                    },
+                    {
+                        "name":"cb_num",
+                        "type":"int"
+                    },
+                    {
+                        "name":"borocode",
+                        "type":"int"
+                    },
+                    {
+                        "name":"boroname",
+                        "type":"char"
+                    },
+                    {
+                        "name":"cncldist",
+                        "type":"int"
+                    },
+                    {
+                        "name":"st_assem",
+                        "type":"int"
+                    },
+                    {
+                        "name":"st_senate",
+                        "type":"int"
+                    },
+                    {
+                        "name":"nta",
+                        "type":"char"
+                    },
+                    {
+                        "name":"nta_name",
+                        "type":"char"
+                    },
+                    {
+                        "name":"boro_ct",
+                        "type":"int"
+                    },
+                    {
+                        "name":"state",
+                        "type":"char"
+                    },
+                    {
+                        "name":"latitude",
+                        "type":"double"
+                    },
+                    {
+                        "name":"longitude",
+                        "type":"double"
+                    },
+                    {
+                        "name":"x_sp",
+                        "type":"double"
+                    },
+                    {
+                        "name":"y_sp",
+                        "type":"double"
+                    }
+                ]
+            },
+            "url":"https://data.cityofnewyork.us/api/views/5rq2-4hqu/rows.csv?accessType=DOWNLOAD"
+        }
+    ],
+    "retriever":"True",
+    "retriever_minimum_version":"2.0.dev",
+    "title":"tree-count, dateset of evert street tree of new-york-city",
```

Comment:
I think "New York City TreesCount" is probably a better title here. The rest belongs in the description where it already is.